libs/collectives/tests/performance/osu/osu_bibw.cpp

```diff
@@ -141,6 +137,6 @@ void run_benchmark(hpx::program_options::variables_map & vm)
 
         double bw = (size / 1e6 * loop * window_size * 2) / t.elapsed();
 
-        hpx::cout << std::left << std::setw(10) << size << bw << std::endl;
+        std::cout << std::left << std::setw(10) << size << bw << std::endl;
```

Comment:
Was this change intentional?

Reply:
Yes, I wanted to remove the dependency on iostreams, the benchmark will print on locality zero only anyways.